client/src/actions/team.js

```diff
@@ -316,3 +344,22 @@ export const updateTeamTheme = (teamId, themeData) => async (dispatch) => {
     dispatch(errorActionGlobalCreator(error));
   }
 };
+
+/**
+ * This action creator will be called when a user want to delete their deployed GitHub Pages
+ *
+ * @param {*} teamId id of the team
+ * @param {*} teamData data object of the data to be deleted
+ * @returns
+ */
+export const deleteGHPages = (teamId, accessToken) => async (dispatch) => {
+  try {
+    const body = {
+      ghToken: accessToken,
+    };
+    await api.deleteGHPages(teamId, body);
+    dispatch(successMessageCreator('The deployed GH Pages have been deleted'));
+  } catch (err) {
+    dispatch(errorActionGlobalCreator(err));
```

Comment:
Related to this comment: https://github.com/Researchify/Researchify/pull/198/files#r711858215
Assuming the user just wanted to reset their data, would they get alarmed at the error toast that says github repo not found?